docs/root/index.md

```diff
@@ -14,25 +14,29 @@ supported here.
 
 ## APIs with supporting libraries
 
-Late beta:
+GA:
 
-- [Google.Cloud.BigQuery.V2](Google.Cloud.BigQuery.V2/index.html)
 - [Google.Cloud.Datastore.V1](Google.Cloud.Datastore.V1/index.html)
 - [Google.Cloud.Logging.V2](Google.Cloud.Logging.V2/index.html)
 - [Google.Cloud.Storage.V1](Google.Cloud.Storage.V1/index.html)
```

Comment:
nit: These are listed in a different order to the list in README.md!! This is terrible, consistency in everything is a must.